nss-ivory-trolls/BangazonAPI
# Cb product controllers

# Description

In addition to the automated tests for the product and productType controllers, I also went ahead and pushed up the controllers and tests for the computer and computerEmployee controllers.

## Type of change


# Testing Instructions for Change Made

- git checkout cb-product-controllers
- please check that the controllers achieve Get All, Get Single, Post, Put and Delete
- please check that the automated tests work for you

# Checklist:
